setup.py

```diff
@@ -16,15 +16,15 @@
     install_requires=[
         "beautifulsoup4==4.7.1",
         "editdistance>=0.5.2, <0.6.0",
-        "emmental==0.0.4",
+        "emmental>=0.0.5, <0.1.0",
         "lxml>=4.2.5, <5.0.0",
         "numpy>=1.11, <2.0",
         "pyyaml>=5.1, <6.0",
         "scipy>=1.1.0, <2.0.0",
-        "snorkel>=0.9.2, <0.10.0",
+        "snorkel @ git+https://github.com/lukehsiao/snorkel.git@luke-patch",
```

Comment:
This one is an attempt to resolve our "dependency hell". Snorkel may eventually update their dependencies, and we can go back to mainline snorkel once they do.

Reply:
Please do not release Fonduer until this line reverts back to the released version of Snorkel.
This is because `https://github.com/lukehsiao/snorkel.git@luke-patch` may not be available for a long-term.

Reply:
Yes, we will not :+1:

Reply:
I don't even want to merge this PR until this line is gone.